wordpress-mobile/WordPress-iOS
# Notifications + Swift Mk 2: App Ratings

#### Details:
This PR adds no new functionality: we're migrating all of the AppRatings-Y code over to Swift (as part of the major effort to rewrite the Notifications stack).

Once it's ready, we'll address code improvements / technical debt.

Reference #5613

#### To test:
1. Hack [this method](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Utility/Ratings/AppRatingUtility.m#L287) so that it always returns true
2. Log into a dotcom account
3. Open the notifications tab

Please, verify that all of the flows behave as expected (Like App > Rate / Dislike App > Feedback).

Needs review: @diegoreymendez 
Thanks in advance!!
